tests/plugins/widget/manual/clipboardhtmldnd.html

```diff
@@ -0,0 +1,60 @@
+<div id="editor">
+	<p>Drop zone</p>
+	<p>[Start here</p>
+	<div data-widget="customwidget">Widget</div>
+	<p>Consectetur dolores voluptatibus illo ipsam eveniet?</p>
+	<div data-widget="customnestedwidget">
+		<p>Widget</p>
+		<div class="ned">
+			<div data-widget="customwidget">Nested widget</div>
+		</div>
+	</div>
+	<p>end here].</p>
+	<p>Some [[inline widget]], too!</p>
```

Comment:
Is there any purpose for this inline widget? :thinking: It doesn't count copies which seems invalid during monkey clicking.